guide/general/faq.md

```diff
@@ -84,3 +96,45 @@ You can also adapt this macro to dismiss the {{site.data.spell.ee}} after gettin
 /cast [talent:6/2] Harden Skin
 /script PetDismiss()
 ```
+
+-T2 Macro - Swaps between {{ site.data.talent.ecs }} and {{ site.data.talent.eb }} depending on selected talent.
+```
+ #showtooltip
+/use [talent:2/2]Echoing Shock;[talent:2/3]Elemental Blast
+ ```
+ 
+ -T5 Macro - Swaps between  {{ site.data.talent.ag }} and {{ site.data.talent.wrt }} depending on selected talent. Casts {{ site.data.talent.wrt }} at your cursor.
+ ```
+ #showtooltip
+/use [talent:2/2]Echoing Shock;[talent:2/3]Elemental Blast
+ ```
+ 
+-T5 Macro - Swaps between :ancestral_guidance: Ancestral Guidance and :wind_rush_totem: Wind Rush Totem depending on selected talent. Casts :wind_rush_totem: Wind Rush Totem at your cursor.
```

Comment:
something went wrong from here on out, either this is a syntax I dont know or its not converted into the link format